### ISSUE TITLE
Mirrored Nexus Repository has broken maven-metadata.xml

### ISSUE THREAD

**Issue by SirAlvarex** [eclipse-ee4j/jersey#4035]
Not sure if this is the right place, but as of this morning (January 22nd, 2019) we can no longer pull any version of jersey-client from the central maven repo.  Our mirror maven-metadata.xml only has this contents:

<metadata>
<groupId>org.glassfish.jersey.core</groupId>
<artifactId>jersey-client</artifactId>
<versioning>
<latest>2.28-RC4</latest>
<release>2.28-RC4</release>
<versions>
<version>2.28-RC4</version>
</versions>
<lastUpdated>20190121195554</lastUpdated>
</versioning>
</metadata>

---

This is the only repo we are having issues building.

**Comment by jansupol** [comment]
There are more Jakarta EE projects having the same issue: 
[HK2](https://github.com/eclipse-ee4j/glassfish-hk2/issues/439)
I have filed a bug on Sonatype, but it is not publicly visible.

**Comment by SirAlvarex** [comment]
Is the HK2 thread you linked the source where we can follow along?

**Comment by jansupol** [comment]
No, just saying there are multiple projects having the same issue. Just go a notice from Sonatype that they started fixing it, but it will take some time. My guess, it will be fixed tomorrow midday/afternoon.

**Comment by jansupol** [comment]
I got a notice from Sonatype that won't be fixed until next week, the impacted data amount is too huge

**Comment by jansupol** [comment]
Seems Jersey is fixed
